SPECS/cloud-init/cloud-init.spec

```diff
@@ -1,16 +1,15 @@
 Summary:        Cloud instance init scripts
 Name:           cloud-init
-Version:        23.3.3
-Release:        2%{?dist}
+Version:        23.4.3
+Release:        1%{?dist}
 License:        GPLv3
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/Base
 URL:            https://launchpad.net/cloud-init
 Source0:        https://github.com/canonical/%{name}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Source1:        10-azure-kvp.cfg
-Patch0:         overrideDatasourceDetection.patch
-Patch1:         make_fallback_network_config_work.patch
```

Comment:
The `overrideDatasourceDetection` patch contains the wrong commit id; it's actually commit 76b7b3814, which is included in version 23.4.

The fallback patch is no longer needed since it applies to the `Photon` distro class, while the new `azurelinux` distro class doesn't inherit from `Photon`.